src/data/index.js

```diff
@@ -17,6 +17,12 @@ export const patterns = [{
 	tags: "date,time"
 },
 {
+	name:"Date and Time format",
+	regex:/^(0[1-9]|[12][0-9]|3[01])([\/])(0[1-9]|1[012])\2([0-9][0-9][0-9][0-9]) ([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]$/,
```

Comment:
@kelvins , instead, maybe you could allow the use of slashes OR dashes to delimit the day, month, and year in the existing implementation of the ISO 8601 in the list.